tests/apis.js

```diff
@@ -526,32 +470,201 @@ test.serial('APIs: Types', t => {
   t.true(isEqual(apiTypes.keys, keys), 'Keys are as expected');
 });
 
+//////////////////////////////
+// APIs - all
+//////////////////////////////
 test.serial('API: All', t => {
   return api.all({}, allTypes).then(results => {
     t.true(results.hasOwnProperty('items'), 'Has Items');
     t.true(results.hasOwnProperty('pages'), 'Has Pagination');
     t.true(results.items.length >= generated, 'Has at least all items in it');
+
+    results.items.forEach(item => {
+      // Ignore empty object
+      if (Object.keys(item).length !== 0) {
+        utils.formatted(t, item);
+      }
+    });
+  });
+});
+
+test.serial('API: All with depth 0', t => {
+  return api.all({
+    depth: 0,
+  }, allTypes).then(results => {
+    let one = false;
+
+    t.true(results.hasOwnProperty('items'), 'Has Items');
+    t.true(results.hasOwnProperty('pages'), 'Has Pagination');
+    t.true(results.items.length >= generated, 'Has at least all items in it');
+
+    results.items.forEach(item => {
+      // Ignore empty object
+      if (Object.keys(item).length !== 0) {
+        one = true;
+        utils.formatted(t, item);
```

Comment:
For instance, for all of these `APIs - all` tests _without_ `follow`, they all should return _identical_ results, because `depth` should only affect this if `follow` is called and the actual content gets generated. It's impossible, without me introspecting each test at run-time, to know if these tests are passing because they are producing the correct results, or passing because of a false-positive from the shared `utils.formatted`